scripts/test-py3.sh

```diff
@@ -25,24 +25,7 @@ pytest openlibrary/mocks openlibrary/olbase openlibrary/utils scripts/tests \
     openlibrary/catalog/merge/test_merge.py \
     openlibrary/catalog/merge/test_merge_marc.py \
     openlibrary/catalog/merge/test_names.py \
-    openlibrary/catalog/merge/test_normalize.py \
-    openlibrary/tests/core/test_cache.py \
-    openlibrary/tests/core/test_connections.py \
-    openlibrary/tests/core/test_helpers.py \
-    openlibrary/tests/core/test_ia.py \
-    openlibrary/tests/core/test_i18n.py \
-    openlibrary/tests/core/test_init.py \
-    openlibrary/tests/core/test_lending.py \
-    openlibrary/tests/core/test_lists_engine.py \
-    openlibrary/tests/core/test_models.py \
-    openlibrary/tests/core/test_olmarkdown.py \
-    openlibrary/tests/core/test_processors.py \
-    openlibrary/tests/core/test_processors_invalidation.py \
-    openlibrary/tests/core/test_ratings.py \
-    openlibrary/tests/core/test_sponsors.py \
-    openlibrary/tests/core/test_vendors.py \
-    openlibrary/tests/core/test_waitinglist.py \
-    openlibrary/tests/solr/test_update_work.py 
+    openlibrary/catalog/merge/test_normalize.py
```

Comment:
Shouldn't the tests directory be added back? Or am I mis-understanding something?

```suggestion
    openlibrary/catalog/merge/test_normalize.py \
    openlibrary/tests
```

Reply:
Yes.  Thanks